telemetry/telemetry.proto

```diff
@@ -23,3 +98,68 @@ message Position {
     float absolute_altitude_m = 3;
     float relative_altitude_m = 4;
 }
+
+message Quaternion {
+    float w = 1;
+    float x = 2;
+    float y = 3;
+    float z = 4;
+}
+
+message EulerAngle {
+    float roll_deg = 1;
+    float pitch_deg = 2;
+    float yaw_deg = 3;
+}
+
+message SpeedNED {
+    float velocity_north_m_s = 1;
+    float velocity_east_m_s = 2;
+    float velocity_down_m_s = 3;
+}
+
+message GPSInfo {
+    int32 num_satellites = 1;
+    FixType fix_type = 2;
+}
+
+enum FixType {
+    NO_GPS = 0;
+    NO_FIX = 1;
+    FIX_2D = 2;
+    FIX_3D = 3;
+    FIX_DGPS = 4;
```

Comment:
Missing **5: RTK float**, **6: RTK fixed**

Reply:
Oh, interesting! I copied a comment in the telemetry plugin. Adding that right away!

Reply:
My bad, I misread the comment actually -_-